tsconfig.eslint.json

```diff
@@ -0,0 +1,9 @@
+{
+  "include": ["src/**/*"],
+  "extends": "./tsconfig.json",
+  "exclude": [
+    "**/node_modules",
+    "**/.*/",
+    "**/lib",
+  ]
```

Comment:
this config is completely the same as `tsconfig.json` - do you think we really need it?